docs/src/main/asciidoc/micrometer-metrics.adoc

```diff
@@ -0,0 +1,283 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Micrometer Metrics
+
+include::./attributes.adoc[]
+
+This guide demonstrates how your Quarkus application can utilize the Micrometer
+metrics library for runtime and application metrics.
+
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* Apache Maven {maven-version}
+
+== Architecture
+
+In this example, we build a very simple microservice which offers one REST endpoint and that determines
+if a number is prime.
+
+== Solution
+
+We recommend that you follow the instructions in the next sections and create the application step by step.
+However, you can go right to the completed example.
+
+Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+
+The solution is located in the `micrometer-quickstart` {quickstarts-tree-url}/micrometer-quickstart[directory].
+
+== Creating the Maven Project
+
+First, we need a new project. Create a new project with the following command:
+
+[source, subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=micrometer-quickstart \
+    -Dextensions="micrometer"
+cd micrometer-quickstart
+----
+
+This command generates a Maven project, that imports the `micrometer` extension and the micrometer
+prometheus registry as dependencies.
```

Comment:
Does this add the prometheus registry dependency automatically? I thought it would just be the micrometer extension only and the prometheus registry needs to be manually added?

Reply:
Not sure if you sure this comment @ebullient?